File: go/raindrops/raindrops.go
package raindrops

import "strconv"

func Convert(input int) string {
  var expected string

  if input % 3 == 0 {
    expected += "Pling"
  }
  if input % 5 == 0 {
    expected += "Plang"
  }
  if input % 7 == 0 {
    expected += "Plong"
  }
  if expected == "" {
    expected += strconv.Itoa(input)
  }

  return expected
}

File: python/raindrops/raindrops.py
def raindrops(number):
    raindrop_speak = ""

    if number % 3 == 0:
        raindrop_speak += "Pling"
    if number % 5 == 0:
        raindrop_speak += "Plang"
    if number % 7 == 0:
        raindrop_speak += "Plong"

    if raindrop_speak == "":
        return str(number)

    return raindrop_speak
